src/IO/H5/Version.cpp

```diff
@@ -19,5 +19,18 @@ Version::Version(bool exists, detail::OpenGroup&& group, hid_t location,
         return version;
       }()),
       group_(std::move(group)) {}
+
+Version::Version(bool exists, detail::OpenGroup&& group, hid_t location,
```

Comment:
This is an overload for writing the version as a string instead of an unsigned int, which is what SpEC does for this type of file.

Reply:
Okay. I guess this can be removed after the catalog is converted? Could you add an explanation to the hpp to the effect of:
```
This is used for fixing a bug in CCE data written by SpEC and will be removed once the CCE data is updated. Do not use for SpECTRE versioning!
```